### ISSUE TITLE
add license link on launch

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2912]
For some reason people keep entering the number wrong in the license form and only see a license link after trying a licensed feature. Show the license link at the end of the launch to make sure people see it and enter the correct number in checkout form

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.47.3](https://github.com/open-wa/wa-automate-nodejs/compare/4.47.2...4.47.3) (2022-10-14)

- chatwoot documentation [`#2885`](https://github.com/open-wa/wa-automate-nodejs/pull/2885)
- ⬆️  upgrade docs deps [`5e294cf`](https://github.com/open-wa/wa-automate-nodejs/commit/5e294cffc442a5c60db5bb3fbfd0b9aa81c963cb)
- 📝  documentation updates [`77d35fe`](https://github.com/open-wa/wa-automate-nodejs/commit/77d35fef4b1c3c78975b42d4a5035589aaca3ab9)
- 📝  fix gr checkout [`19e7829`](https://github.com/open-wa/wa-automate-nodejs/commit/19e7829154acffe05d5d777312a1ad2448a94fbb)
- Release 4.47.3 [`0a171a1`](https://github.com/open-wa/wa-automate-nodejs/commit/0a171a11eab7b98f3a7cd63da5a98a10f18d2b2c)
- 🔊 add lic link to logs to prevent bad form entry #2312 [`af94734`](https://github.com/open-wa/wa-automate-nodejs/commit/af947342b7c9f4e5849127f4967f910e4c0b2d8b)
- 🥚 patches update [`3153136`](https://github.com/open-wa/wa-automate-nodejs/commit/3153136a7a961411926291e55282f6ee58056865)
- updated types-only package [`e80e534`](https://github.com/open-wa/wa-automate-nodejs/commit/e80e53498d90a6506e4e3afcfc5fff4ccef2b628)
